main.go

```diff
@@ -13,29 +14,76 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var (
+	baseCovLinkTemplate = flag.String("link-template.base", "", "template used to generate links to base commit coverage profiles. See LINK TEMPLATES")
+	headCovLinkTemplate = flag.String("link-template.head", "", "template used to generate links to head commit coverage profiles. See LINK TEMPLATES")
+
+	dryrun = flag.Bool("dry-run", false, "skip posting comment to Github, printing to stdout instead")
+)
+
+func init() {
+	flag.Usage = func() {
+		out := flag.CommandLine.Output()
+		fmt.Fprintf(out, "Usage: %s [OPTIONS] <base-profile> <head-profile>\n", os.Args[0])
+
+		fmt.Fprintln(out, "\nOPTIONS")
+		flag.PrintDefaults()
+
+		fmt.Fprintln(out, "\nLINK TEMPLATES")
+		fmt.Fprintln(out, "In the comment posted to Github PR can optionally include links to")
+		fmt.Fprintln(out, "the html version of the cover profile with color coded lines. In")
+		fmt.Fprintln(out, "order to generate these links pass two links templates. The")
+		fmt.Fprintln(out, "following tokens will be replaced in the template:")
+		fmt.Fprintln(out, "")
+		fmt.Fprintln(out, "\t%[p] -> the current go package name")
+		fmt.Fprintln(out, "")
+		fmt.Fprintln(out, "Example")
+		fmt.Fprintln(out, "$ goverdiff \\")
+		fmt.Fprintln(out, "\t-link-template.base 'https://ci.acme.com/coverprofiles/before/%[p]/coverage.html' \\")
+		fmt.Fprintln(out, "\t-link-template.head 'https://ci.acme.com/coverprofiles/after/%[p]/coverage.html' \\")
+		fmt.Fprintln(out, "\t base/coverage.out head/coverage.out")
+	}
+}
+
 func main() {
 	ctx := context.Background()
+	flag.Parse()
+
+	if *baseCovLinkTemplate != "" {
+		if *headCovLinkTemplate == "" {
+			fmt.Fprintln(os.Stderr, "fatal: expected link-template.head option to not be empty")
+			os.Exit(1)
+		}
+	} else if *headCovLinkTemplate != "" {
+		fmt.Fprintln(os.Stderr, "fatal: expected link-template.base option to not be empty")
+		os.Exit(1)
+	}
 
 	// load given base and head `go test` cover profiles from disk
-	base, err := LoadCoverProfile(os.Args[1])
+	base, err := LoadCoverProfile(flag.Arg(0))
 	if err != nil {
 		panic(err)
 	}
 
-	head, err := LoadCoverProfile(os.Args[2])
+	head, err := LoadCoverProfile(flag.Arg(1))
 	if err != nil {
 		panic(err)
 	}
 
 	// generate and publish GitHub pull request message
-	createOrUpdateComment(
-		ctx,
-		summaryMessage(base.Coverage(), head.Coverage()),
-		buildTable(getModulePackageName(), base, head))
+	sum := summaryMessage(base.Coverage(), head.Coverage())
+	tab := buildTable(getModulePackageName(), base, head)
+
+	if *dryrun {
+		fmt.Println(sum)
+		fmt.Println(tab)
+	} else {
+		createOrUpdateComment(ctx, sum, tab)
+	}
 }
 
 func buildTable(rootPkgName string, base, head *CoverProfile) string {
-	const tableRowSprintf = "%-80s %8s %8s %8s\n"
+	const tableRowSprintf = "|%-80s|%8s|%8s|%8s|\n"
```

Comment:
I do like the idea of the package name being in a fixed width/code font.

```suggestion
	const tableRowSprintf = "|`%-78s`|%8s|%8s|%8s|\n"
```

Reply:
Also wondering if the percentages should be fixed width too?

```suggestion
	const tableRowSprintf = "|%-80s|`%6s`|`%6s`|`%6s`|\n"
```

Lets see how you go.